linux_os/guide/system/permissions/files/file_permissions_ungroupowned/oval/shared.xml

```diff
@@ -8,22 +8,40 @@
   </definition>
 
   <!-- Create a file_state to filter out files group-owned by known groups. -->
-  <ind:textfilecontent54_object id="etc_group_objects" version="1">
+  <ind:textfilecontent54_object id="object_etc_group" version="1">
     <ind:filepath>/etc/group</ind:filepath>
     <ind:pattern operation="pattern match">^[^:]+:[^:]*:([\d]+):[^:]*$</ind:pattern>
     <ind:instance operation="greater than or equal" datatype="int">1</ind:instance>
   </ind:textfilecontent54_object>
 
+  <ind:textfilecontent54_object id="object_usr_lib_group" version="1">
+    <ind:filepath>/usr/lib/group</ind:filepath>
```

Comment:
It would be great to not read this file always but to read this file only if the nsswitch.conf is configured to use altfiles for groups. How can I implement this condition in OVAL?

Reply:
yes